source/renderer/app/stores/SidebarStore.js

```diff
@@ -62,6 +76,57 @@ export default class SidebarStore extends Store {
     });
   }
 
+  _sortWallets = (wallets: Wallet[]): Wallet[] => {
+    const { sortBy, sortOrder } = this.walletSortConfig;
+
+    type IndexedWallet = { wallet: Wallet, index: number };
+
+    const walletsWithIndex: IndexedWallet[] = wallets.map(
+      (w: Wallet, index: number) => ({ wallet: w, index })
+    );
+
+    const amountWeight = ({ wallet }: IndexedWallet) =>
```

Comment:
1. micro-optimization but all those getters / pickers are recreated with each `_sortWallets` call.
2. can we name them like a functions instead of values e.g. `getAmountWeight` or `pickAmountWeight` ?